doc/module-docs/cc_package_update_upgrade_install/data.yaml

```
@@ -9,5 +9,9 @@ cc_package_update_upgrade_install:
   - comment: |
       Example 1:
     file: cc_package_update_upgrade_install/example1.yaml
+  - comment: "By default, `package_upgrade: true` will trigger upgrades on any installed package manager. To avoid calling `snap refresh` in images with snap installed, setting snap refresh.hold to a time limit to prevent cloud-init interaction during first boot"
+    file: cc_package_update_upgrade_install/example2.yaml
+  - comment: "By default, `package_upgrade: true` will trigger upgrades on any installed package manager. To avoid calling `snap refresh` in images with snap installed, setting snap refresh.hold to forever will prevent cloud-init interaction during any boot"
```

```suggestion
  - comment: "By default, `package_upgrade: true` will trigger upgrades on any installed package manager. To avoid calling `snap refresh` in images with snap installed, set snap refresh.hold to forever will prevent cloud-init interaction during any boot"
```

see previous comment